modules/sttp/src/main/scala/com/caesars/tracing/TracedBackend.scala

```diff
@@ -0,0 +1,58 @@
+package com.caesars.tracing
+
+import io.janstenpickle.trace4cats.{ErrorHandler, ToHeaders}
+import io.janstenpickle.trace4cats.model.{SampleDecision, SpanKind}
+import io.janstenpickle.trace4cats.sttp.client3.{SttpRequest, SttpSpanNamer}
+import io.janstenpickle.trace4cats.sttp.common.{SttpHeaders, SttpStatusMapping}
+import sttp.capabilities.Effect
+import sttp.client3.impl.zio.RIOMonadAsyncError
+import sttp.client3.*
+import sttp.monad.MonadError
+import zio.*
+
+object TracedBackend {
+  final case class TracedBackendConfig(
+      spanNamer: SttpSpanNamer = SttpSpanNamer.methodWithPath,
```

Comment:
Trace Span Names should have low or high cardinality? 

I ask since, with this `SttpSpanNamer.methodWithPath`, a span's name will be `$requestMethod $requestPath`. So, if the request's path includes a dynamic field, e.g. `GET /foo/{id}`, the cardinality will be high.

Reply:
This is a good question - I looked into Trace4Cats implementation for HTTP4S and saw they follow this convention: 
https://opentelemetry.io/docs/reference/specification/metrics/semantic_conventions/http-metrics/#parameterized-attributes - so they want low cardinality

Reply:
Nice link, Calvin. Given that we want low cardinality, this `SttpSpanNamer.methodWithPath` isn't a reasonable default, no?

Perhaps we want a classifier, i.e. something like [this](https://github.com/http4s/http4s/blob/series/0.23/core/shared/src/main/scala/org/http4s/metrics/MetricsOps.scala#L77-L110), for building a low cardinality path?